### PR TITLE
Skip equity tier limit check in PlaceShortTermOrder

### PR DESCRIPTION
**Pull request by roy-dydx**
### Changelist
These checks are not effective for their original purpose and take 30% of the CheckTx global lock time in testnet loads.

### Test Plan
image

### Author/Reviewer Checklist
- [ ] If this PR has changes that result in a different app state given the same prior state and transaction list, manually add the `state-breaking` label.
- [ ] If the PR has breaking postgres changes to the indexer add the `indexer-postgres-breaking` label.
- [ ] If this PR isn't state-breaking but has changes that modify behavior in `PrepareProposal` or `ProcessProposal`, manually add the label `proposal-breaking`.
- [ ] If this PR is one of many that implement a specific feature, manually label them all `feature:[feature-name]`.
- [ ] If you wish to for mergify-bot to automatically create a PR to backport your change to a release branch, manually add the label `backport/[branch-name]`.
- [ ] Manually add any of the following labels: `refactor`, `chore`, `bug`.

### CODE REVIEW

File: protocol/x/clob/keeper/orders.go [hunk 1]
```diff
@@ -208,12 +208,6 @@ func (k Keeper) PlaceShortTermOrder(
 		return 0, 0, err
 	}
 
-	// Validate that adding the order wouldn't exceed subaccount equity tier limits.
-	err = k.ValidateSubaccountEquityTierLimitForShortTermOrder(ctx, order)
-	if err != nil {
```
Comment (jonfung-dydx) [inline protocol/x/clob/keeper/orders.go:213]: remove the comment on line 170

Comment (jonfung-dydx) [reply protocol/x/clob/keeper/orders.go:213]: we also technically dont use `ValidateSubaccountEquityTierLimitForShortTermOrder` anymore maybe add a comment that we removed the sa eq tier check for short term orders on check tx to that fn or just del

**Review by jonfung-dydx — APPROVED** [review]

File: protocol/x/clob/keeper/orders.go (hunk 1 above)
Comment (roy-dydx) [reply protocol/x/clob/keeper/orders.go:213]: OK I marked it as deprecated.

**Comment by roy-dydx** [comment]
@Mergifyio backport release/protocol/v4.1.x

**Pull request merged**